java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/api/v1/menu/dto/requestDto/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,70 @@
+package com.xxAMIDOxx.xxSTACKSxx.api.v1.menu.dto.requestDto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+/**
+ * A DTO to map Create Category request.
+ *
+ * @author ArathyKrishna
+ */
+
+@Schema(name = "CreateCategoryRequest")
+public class CreateCategoryRequestDto {
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonPropertyOrder({
+          "name",
+          "description"
+  })
```

Comment:
Is the above needed when there's already validation on the attributes themselves?